_posts/2022-08-10-#25720.md

```diff
@@ -10,15 +10,53 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- On startup, if a node's headers chain is more than a day behind the current
+  time, it will pick one peer to sync headers with until its best headers chain
+  is caught up. (See
+  [here](https://github.com/bitcoin/bitcoin/blob/92f6461cfd39fff2fc885dd623fa47e7d8d53827/src/net_processing.cpp#L4889-L4913))
+  If a node's headers chain is not caught up within `15 min + ~num headers
+  missing * 1ms` (either because the chosen peer was not able to or because
+  they purposefully stalled the download), it will disconnect the chosen peer
```

Comment:
```suggestion
  they purposefully stalled the download), it will [disconnect](https://github.com/bitcoin/bitcoin/blob/92f6461cfd39fff2fc885dd623fa47e7d8d53827/src/net_processing.cpp#L5223-L5254) the chosen peer
```